clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -475,6 +475,10 @@ raw2trace_t::process_marker(raw2trace_thread_data_t *tdata,
                             trace_marker_type_t marker_type, uintptr_t marker_val,
                             byte *&buf, DR_PARAM_OUT bool *flush_decode_cache)
 {
+    if (marker_type == TRACE_MARKER_TYPE_MIDBLOCK_END_PC) {
+        // Consumed by raw2trace and not made visible in final trace.
```

Comment:
Document cases where this may happen, as to a first time reader it seems to violate the basic principle of basic blocks: that it can be relied on to execute till the bb end (though we did have the kernel_event case from before too). Also worth a similar comment by offline_entry_t.pc.instr_count.